client/src/App.tsx

```diff
@@ -28,6 +29,8 @@ function App(): JSX.Element {
                 <Route exact path="/signup" component={Signup} />
                 <Route exact path="/dashboard" component={Dashboard} />
                 <Route path="/profile/settings" component={Settings} />
+                <Route path="/profile/listing" component={ProfileListing} />
```

Comment:
The dashboard will be where we list the profiles, would make more sense to do the changes in that component rather than a new route.